MediaBrowser.Controller/Net/BasePeriodicWebSocketListener.cs

```diff
@@ -232,6 +232,11 @@ private void DisposeConnection(Tuple<IWebSocketConnection, CancellationTokenSour
                 // TODO Investigate and properly fix.
                 Logger.LogError(ex, "Object Disposed");
             }
+            catch (Exception ex)
+            {
+                // TODO Investigate and properly fix.
+                Logger.LogError(ex, "Object Disposed Exception");
```

Comment:
```suggestion
                Logger.LogError(ex, "Error disposing websocket");
```

We don't know if the exception is because the object was disposed or some other reason